src/Orchard.Cms.Web/Modules/Orchard.Demo/Orchard.Demo.csproj

```diff
@@ -20,6 +20,7 @@
     <ProjectReference Include="..\..\..\OrchardCore\Orchard.Environment.Navigation\Orchard.Environment.Navigation.csproj" />
     <ProjectReference Include="..\..\..\OrchardCore\Orchard.ResourceManagement\Orchard.ResourceManagement.csproj" />
     <ProjectReference Include="..\..\..\OrchardCore\Orchard.ResourceManagement.Abstractions\Orchard.ResourceManagement.Abstractions.csproj" />
+    <ProjectReference Include="..\Orchard.Contents\Orchard.Contents.csproj" />
```

Comment:
I needed access to Orchard.Contents.Permissions.ViewContent . It is declared at this module.